lib/model.js

```diff
@@ -44,9 +45,22 @@ var EventEmitter = require('eventemitter3'),
      * @prop {object} [data] initial data
      */
     Model = Inheritable.inherit({
+
+        /**
+         * @deprecated use getId method
```

Comment:
+1 for deprecate. id should be synchronized in both directions with id attribute, and it costs us in code in models, that should be maintained (at least standard amend in id attribute and, setter for id field)
Also it makes possible to do same things in different ways. Don't think that it is good, it makes interface more complicated.